features/custom-user-agent.json

```
@@ -0,0 +1,19 @@
+{
+    "_meta": {
+        "description": "Custom user agents used by our native apps that override the default user-agent string.",
+        "sampleExcludeRecords": {
+            "domain": "example.com",
+            "reason": "Site Breakage"
```

```suggestion
            "reason": "Site doesn't recognize our user agent and blocks user from logging in."
```

sorry to spend so much time on this :) but I think a good example can be actually useful. I think google auth blocked us in the past so I tried to base the example on that experience